tasks/config_db.yaml

```
@@ -1,5 +1,6 @@
 ---
 - name: Configure MariaDB (my.cnf)
+  when: mariadb.galera.arbitrator == False
```

Raw Jinja2 expression should use lower case `true`/`false`/`none`: https://jinja.palletsprojects.com/en/2.11.x/templates/#literals
```suggestion
  when: mariadb.galera.arbitrator == false
```